tests/integration/targets/ldap_search/tasks/run-test.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Not related to licensing at all, but is this task file suppose to be empty?

Reply:
Fixed in #5077.